<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PC Client Block</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_ip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.36</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>51015</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_listen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Listen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Interval (sec)l:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_interval">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_config">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuration mode</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_sensors">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>161</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Power state</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mouse Activity</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Keyboard activity</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Activity direction</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Internet speed</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Active sensors:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
